lib/rubocop/cop/salsify/style_dig.rb

```diff
@@ -50,7 +50,7 @@ def autocorrect(node)
           lambda do |corrector|
             range = Parser::Source::Range.new(node.source_range.source_buffer,
                                               root_node.source_range.end_pos,
-                                              node.source_range.end_pos)
+                                              node.source_range.end_pos,)
```

Comment:
It's ironic that the major benefit of trailing-commas is hard to realize when it's used with same-line-parens (or brackets). Oh well, better than before...

Reply:
This is what makes me think we should switch from `consistent_comma` to `comma`.